### PR TITLE
fix typos in cname template and better wording

### PR DESCRIPTION
**Pull request by icehess**
Also fixe a typo in 'Sent from' paragraph style.

### CODE REVIEW

File: applications/teletype/priv/templates/cnam_request.html [hunk 1]
```diff
@@ -37,14 +37,14 @@ <h2 style="margin:0;padding:0;text-align:center;font-family:'Open Sans',sans-ser
                 </tr>
                 <tr>
                     <td bgcolor="#ffffff" style="padding:20px;font-family:'Open Sans',sans-serif;color:#555555;">
-                        <p style="margin:0;padding:0;font-family:'Open Sans',sans-serif;color:#555555;">A request to change the Caller name to <b>{{cnam.display_name}}</b> for number <b>{{request.number}}</b> has been submitted. Following are the information about this request.</p>
+                        <p style="margin:0;padding:0;font-family:'Open Sans',sans-serif;color:#555555;">Caller name identification for number <b>{{request.number}}</b> has be changed to <b>{{cnam.display_name}}'</b>. Following is the detailed information about this change.
```
Comment (fenollp) [inline applications/teletype/priv/templates/cnam_request.html]: s `has be changed to` --> `has been changed to`

File: applications/teletype/priv/templates/cnam_request.html [hunk 2]
```diff
@@ -67,7 +67,7 @@ <h3 style="margin:0;padding:0;font-family:'Open Sans',sans-serif;color:#555555;f
                             </tr>
                             <tr>
                                 <td style="padding:10px;background-color:#f5f5f5;text-align:right;font-family:'Open Sans',sans-serif;color:#555555;font-size:13px;font-weight:400;line-height:20px;border-size:3px;border-color:#ffffff;border-right-style:solid;border-bottom-style:solid;" width="30%"><b>Number State</b></td>
-                                <td style="padding:10px;background-color:#eaeaea;font-size:13px;font-family:'Open Sans',sans-serif;color:#555555;line-height:20px;border-size:3px;border-color:#ffffff;border-bottom-style:solid;" width="70%">{{request.state}}</td>
+                                <td style="padding:10px;background-color:#eaeaea;font-size:13px;font-family:'Open Sans',sans-serif;color:#555555;line-height:20px;border-size:3px;border-color:#ffffff;border-bottom-style:solid;" width="70%">{{request.number_state}}</td>
```
Comment (fenollp) [inline applications/teletype/priv/templates/cnam_request.html]: `pvt_number_state` is the old field that is now renamed to `pvt_state` (don't ask me why). Maybe we should read both?

Comment (icehess) [inline applications/teletype/priv/templates/cnam_request.html]: Number_state is the field name for notify.

https://github.com/2600hz/kazoo/blob/master/core/kazoo_number_manager/src/providers/knm_cnam_notifier.erl#L150

https://github.com/2600hz/kazoo/blob/master/core/kazoo_number_manager/src/providers/knm_telnyx_cnam.erl#L155

https://github.com/2600hz/kazoo/blob/master/core/kazoo_number_manager/src/providers/knm_vitelity_cnam.erl#L338

File: applications/teletype/priv/templates/cnam_request.html (hunk 1 above)
Comment (icehess) [inline applications/teletype/priv/templates/cnam_request.html]: hehe, it was supposed to be a fix typo pr :)

**Pull request merged**